cscs-checks/system/openstack/src/s3_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+virtualenv s3virtenv
```

Comment:
This is deprecated from Python 3.6. Check [here](https://docs.python.org/3/library/venv.html). You should create virtual environments as follows:

```suggestion
python3 -m venv s3virtenv
```